kubernetes-templates/04_cvat_frontend_deployment.yml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cvat-frontend
+  namespace: cvat
+  labels:
+    app: cvat-app
+    tier: frontend
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: cvat-app
+      tier: frontend
+  template:
+    metadata:
+      labels:
+        app: cvat-app
+        tier: frontend
+    spec:
+      containers:
+        - name: cvat-frontend-app-container
+          image: openvino/cvat_ui:v1.2.0
+          imagePullPolicy: Always
+          env:
+          - name: REACT_APP_API_PROTOCOL
+            value: "http"
+          - name: REACT_APP_API_HOST
+            value: "{MY_SERVER_URL_COM}"
+          - name: REACT_APP_API_PORT
+            value: "8080"
```

Comment:
These variables are no longer needed and have no effect.

Reply:
Removed these lines in 590d7f0